content/geoip/license-key-validation-api.mdx

```diff
@@ -0,0 +1,97 @@
+---
+draft: false
+title: License Key Validation API
+---
+
+MaxMind provides a License Key Validation API, a tool for developers and
+organizations seeking to verify the validity of MaxMind license keys within
+their systems. This API endpoint is designed to check the existence and format
+of a license key, but it does not provide information about the license key's
+access to MaxMind products or the account to which it is associated.
+
+This API can be particularly useful when integrated with secret scanning tools
+used for compliance purposes. By using our License Key Validation API in
+conjunction with these tools, you can automate the process of finding and
+verifying MaxMind license keys in your codebase. This can help prevent license
+keys accidentally ending up in unexpected places and enhance the security of
+your applications.
+
+## Request:
```

Comment:
How about "Request"? I noticed the ":" shows in the table of contents and looks off to me there.

Reply:
I agree 👍